examples/todo/test/acceptance/application.acceptance.ts

```diff
@@ -50,17 +38,34 @@ describe('Application', () => {
       .post('/todos')
       .send(todo)
       .expect(200);
+    expect(response.body).to.containDeep(todo);
+    const result = await todoRepo.findById(response.body.id);
+    expect(result).to.containDeep(todo);
+  });
+
+  it('creates an address-based reminder', async () => {
```

Comment:
would adding a test for retrieval of a todo with `remindAtGeo` and `remindAtAddress` set add value to our test suite?

Reply:
It's to see that we can retrieve them as well as create them with those fields, but I don't think it's needed tbh.